android/fastlane/Fastfile

```diff
@@ -41,6 +41,13 @@ platform :android do
       track: "internal",
       skip_upload_changelogs: true,
       skip_upload_images: true,
+      release_status: "completed",
+      # Add health-related declarations
+      app_store_metadata: {
```

Comment:
this is for appstore and it is in the android config file... what error are you getting?

Reply:
Yes, exactly. reverted.